fusor-ember-cli/app/styles/custom.scss

```diff
@@ -321,3 +321,7 @@ table.fusor-table {
   padding:0!important;
   cursor: pointer;
 }
+
+.eye-icon {
+  /* TODO */
```

Comment:
@isratrade, one way to do this would be to absolutely position the icon.  make sure that the *parent* of input.form-control and .eye-icon is positioned (e.g., position: relative) and you add padding to the input.form-control on the right side to allow space for the icon.